### PR TITLE
IPU: refine the commands when doing leapp test

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID:2567

Refined the commands when doing in place upgrade

### CODE REVIEW

**Comment by MiriamDeng** [comment]
@fbq815 @PaulYuuu could you give a hand to review this patch ?  Thank you !

**Comment by fbq815** [comment]
(1/2) Host_RHEL.m8.u10.product_rhel.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.10.0.s390x.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.s390-virtio: PASS (633.08 s)
 (2/2) Host_RHEL.m8.u10.product_rhel.nographic.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.10.0.s390x.io-github-autotest-qemu.in_place_upgrade.without_rhsm_rhel8_to_rhel9.compose_install.s390-virtio: PASS (583.91 s)

LGTM

**Review by fbq815 — APPROVED** [review]

File: qemu/tests/cfg/in_place_upgrade.cfg [hunk 1]
```diff
@@ -59,7 +59,9 @@
         - without_rhsm_rhel8_to_rhel9:
             rhsm_type = "no_rhsm"
             pre_upgrade_no_rhsm = "leapp preupgrade --debug --no-rhsm"
-            process_upgrade_no_rhsm = "leapp upgrade --debug --no-rhsm"
+            process_upgrade_no_rhsm_1 = "export LEAPP_UNSUPPORTED=1"
+            process_upgrade_no_rhsm_2 = "export LEAPP_DEVEL_SKIP_CHECK_OS_RELEASE=1"
+            process_upgrade_no_rhsm_3 = "leapp upgrade --debug --no-rhsm"
```
Comment (PaulYuuu) [inline qemu/tests/cfg/in_place_upgrade.cfg:64]:
```suggestion
            process_upgrade_no_rhsm = "LEAPP_UNSUPPORTED=1 LEAPP_DEVEL_SKIP_CHECK_OS_RELEASE=1 leapp upgrade --debug --no-rhsm"
```

Comment (MiriamDeng) [reply qemu/tests/cfg/in_place_upgrade.cfg:64]: Sure, will update it  soon.

File: qemu/tests/cfg/in_place_upgrade.cfg [hunk 2]
```diff
@@ -59,7 +59,9 @@
         - without_rhsm_rhel8_to_rhel9:
             rhsm_type = "no_rhsm"
             pre_upgrade_no_rhsm = "leapp preupgrade --debug --no-rhsm"
-            process_upgrade_no_rhsm = "leapp upgrade --debug --no-rhsm"
+            process_upgrade_no_rhsm = "export LEAPP_UNSUPPORTED=1;"
+            process_upgrade_no_rhsm += "export LEAPP_DEVEL_SKIP_CHECK_OS_RELEASE=1;"
+            process_upgrade_no_rhsm += "leapp upgrade --debug --no-rhsm"
```
Comment (PaulYuuu) [inline qemu/tests/cfg/in_place_upgrade.cfg:64]: I think LEAPP_xxx will only be used once, so not need to export them, just use env variables and pass to the command line, like in my previous comments.

**Review by PaulYuuu — DISMISSED** [review]

**Comment by MiriamDeng** [comment]
Test results
 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.10.0.x86_64.io-github-autotest-qemu.in_place_upgrade.without_rhsm_rhel8_to_rhel9.stream_install.q35: STARTED
 (1/1) Host_RHEL.m9.u5.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.10.0.x86_64.io-github-autotest-qemu.in_place_upgrade.without_rhsm_rhel8_to_rhel9.stream_install.q35: PASS (2772.47 s)

**Comment by PaulYuuu** [comment]
The commit should have gpg signed.

**Comment by MiriamDeng** [comment]
Updated, thanks. Can you help to review ?

**Comment by PaulYuuu** [comment]
> Updated, thanks. Can you help to review ? @PaulYuuu

Can you double check? I don't see the commit being signed.

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**